op-deployer/pkg/deployer/upgrade/v200/upgrade.go

```diff
@@ -0,0 +1,61 @@
+package v200
```

Comment:
could we rename this package `v200` --> `v2_0_0` to preserve the semver a bit better?